conf/modules.config

```diff
@@ -119,6 +125,10 @@ process {
         ext.suffix = "tsv"
     }
 
+    withName: "RUN_DYNAMITE" {
+        errorStrategy = "ignore"
+    }
```

Comment:
In what situations is this relevant? By setting the `errorStrategy` to `ignore`, we also prevent the pipeline from trying again if it fails due to too little RAM etc